targets/xiwi

```diff
@@ -402,7 +403,13 @@ EOF
     /usr/bin/install -s dummy_drv.so "$DRIVERDIR/"
 ) # End compilation subshell
 
-if [ "${DISTROAKA:-"$DISTRO"}" = 'debian' ]; then
+if [ "${DISTROAKA:-"$DISTRO"}" = 'openSUSE' ]; then
+    if [ $ARCH = "aarch64" ] || [ $ARCH = "x86_64"]; then
+        cp /usr/lib/xorg/modules/drivers/* /usr/lib64/xorg/modules/drivers/
```

Comment:
ok, I see, I should change the DRIVEDIR variable accordingly. I just wonder if on ubuntu/debian etc the drivers are installed under /usr/lib/... although the architecture is amd64. Can you confirm? Otherwise we could get rid of the first if, checking for openSUSE